Error when getting LAN address without netifaces

I was running some Tribler tests on a machine without `netifaces` installed, and I got the following error:

```
self = <ipv8.peerdiscovery.community.DiscoveryCommunity object at 0x7f292d4b37b8>
bootstrap = True

    def _get_lan_address(self, bootstrap=False):
        """
        Attempt to get the newest lan ip of this machine, preferably with netifaces, but use the fallback if it fails
        :return: lan address
        """
        if self._netifaces_failed:
>           return (self.get_lan_address_without_netifaces(), self.endpoint._port)
E           AttributeError: 'DispatcherEndpoint' object has no attribute '_port'

src/pyipv8/ipv8/messaging/interfaces/endpoint.py:290: AttributeError
```

Also see [here](https://jenkins-ci.tribler.org/job/validation_experiments/job/validation_experiment_libtorrent_compatibility/601/console).

--- a/ipv8/messaging/interfaces/endpoint.py
+++ b/ipv8/messaging/interfaces/endpoint.py
@@ -287,12 +287,12 @@ class EndpointListener(metaclass=abc.ABCMeta):
         :return: lan address
         """
         if self._netifaces_failed:
-            return (self.get_lan_address_without_netifaces(), self.endpoint._port)
+            return self.get_lan_address_without_netifaces(), self.endpoint.get_address()[1]
         else:
             self._local_interfaces = list(self._get_interface_addresses())
             interface = self._guess_lan_address(self._local_interfaces)
             return (interface.address if interface else self.get_lan_address_without_netifaces()), \
-                   (0 if bootstrap else self.endpoint._port)
+                   (0 if bootstrap else self.endpoint.get_address()[1])
 
     def _guess_lan_address(self, interfaces, default=None):
         """

